<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GE_WidgetClass</class>
 <widget class="QWidget" name="GE_WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GE_Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="MyScrollArea" name="scrollArea_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>573</width>
          <height>730</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>300</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>300</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(202, 202, 202);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Brightless</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Exposure Auto</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Continuous</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Once</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Off</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>ROI</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>374</width>
              <height>317</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Image Format</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Pixel Format:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" colspan="3">
                 <widget class="QComboBox" name="comboBox">
                  <item>
                   <property name="text">
                    <string>Mono8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Mono12</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Mono12Packed</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Binning Horizontal:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QSpinBox" name="spinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>8</number>
                  </property>
                  <property name="value">
                   <number>6</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Binning Vertical:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="4">
                 <widget class="QSpinBox" name="spinBox_2">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>11</number>
                  </property>
                  <property name="value">
                   <number>6</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="5">
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Preferred</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>67</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>ROI</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="1" column="0">
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="title">
                   <string>GroupBox</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="1" column="3">
                    <widget class="QSpinBox" name="spinBox_6">
                     <property name="maximum">
                      <number>4872</number>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="2">
                    <widget class="QPushButton" name="pushButton_5">
                     <property name="text">
                      <string>1/8</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QPushButton" name="pushButton_4">
                     <property name="text">
                      <string>1/4</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QSpinBox" name="spinBox_3">
                     <property name="maximum">
                      <number>3248</number>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="3">
                    <widget class="QPushButton" name="pushButton_9">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>../../ms_qt/GE_Widget/photo.png</normaloff>../../ms_qt/GE_Widget/photo.png</iconset>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>61</width>
                       <height>20</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QSpinBox" name="spinBox_5">
                     <property name="maximum">
                      <number>3248</number>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>OffsetX</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QSpinBox" name="spinBox_4">
                     <property name="maximum">
                      <number>4872</number>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="2">
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>Width</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QPushButton" name="pushButton_3">
                     <property name="text">
                      <string>Full</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>OffsetY</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>Height</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="MyScrollArea" name="scrollArea_3">
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents_3">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>274</width>
                     <height>274</height>
                    </rect>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="Widget" name="widget" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>256</width>
                        <height>256</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>256</width>
                        <height>256</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../ms_qt/GE_Widget/play-button.jpg</normaloff>../../ms_qt/GE_Widget/play-button.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../ms_qt/GE_Widget/stop-button.jpg</normaloff>../../ms_qt/GE_Widget/stop-button.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header>widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyScrollArea</class>
   <extends>QScrollArea</extends>
   <header>myscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GE_Widget.qrc"/>
 </resources>
 <connections/>
</ui>
